internal/engine/k8swatch/reducers.go

```diff
@@ -101,7 +101,14 @@ func maybeUpdateStateForPod(ms *store.ManifestState, pod *v1alpha1.Pod) bool {
 	// Track the new version
 	runtime.Pods[podID] = pod
 
-	if len(pod.Containers) != 0 && (store.AllPodContainersReady(*pod) || pod.Phase == string(v1.PodSucceeded)) {
+	isReadyOrSucceeded := false
+	if ms.K8sRuntimeState().PodReadinessMode == model.PodReadinessComplete {
```

Comment:
the model is "Complete" but the condition is "Succeeded".

Are we assuming "Complete" and "Succeeded" are interchangeable here?

(I know airflow calls this ALL_DONE vs ALL_SUCCESS - https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html?highlight=all_done#trigger-rules)

Reply:
good point! changed model to "Succeeded"